api/HTMLDocument.json

```diff
@@ -0,0 +1,820 @@
+{
+  "api": {
+    "HTMLDocument": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLDocument",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
```

Comment:
HTMLDocument is still implemented in Firefox, although some of the members have migrated to Document already. This should be true for both Firefox and Firefox Android throughout this file (unless it's decided to hold that off for another update).